src/shared/palette/complex/write-complex-field.component.spec.ts

```diff
@@ -165,12 +165,12 @@ describe('WriteComplexFieldComponent', () => {
       fixture.detectChanges();
     }));
 
-    it('should not render a form group with a header for the complex type', () => {
+    it('should render a form group with a header for the complex type', () => {
       let panelTitle = de
         .query($COMPLEX_PANEL)
         .query($COMPLEX_PANEL_TITLE);
 
-      expect(panelTitle).toBeNull();
+      expect(panelTitle).not.toBeNull();
```

Comment:
Yes I noticed this tests was broken for a while. Actually there used to be one more expect in here:
`expect(panelTitle.nativeElement.textContent).toBe(CASE_FIELD.label + ' (Optional)');`

Reply:
https://github.com/hmcts/ccd-case-management-web/commit/044250fc8b29cf682d5c95fb8e57b52d797acde4#diff-a86ccedb6ea82f86e6fd8ba8df1672bf

Reply:
Ok updated that now in V's absence.